### ISSUE TITLE
fix(bal-shape): Hide Shape from Screen Reader

### ISSUE THREAD

**Issue by GinaBiondo** [baloise/design-system#983]
### Current Behavior

Currently the screen reader tries to read information from the shape.

### Expected Behavior

Purely decorative SVGs like the shape should be hidden from the screen reader

### Steps to Reproduce

Regular usage

### Version

13.4.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
